core/src/main/kotlin/gropius/model/architecture/Trackable.kt

```diff
@@ -66,7 +66,7 @@ abstract class Trackable(
     @property:GraphQLDescription("If existing, the URL of the repository (e.g. a GitHub repository).")
     @FilterProperty
     var repositoryURL: URI?
-) : AffectedByIssue(name, description) {
+) : AffectedByIssue(name, description){
```

Comment:
reformat or reformat everything else to reach consistent formatting

Reply:
reformatted with 7059692890af6147fa7e7347d01b098470041107